### PR TITLE
Origin/add flags skip data load

### PR DESCRIPTION
**Pull request by sandeepmanocha**
Pull request to add feature to re-create views without re-loading the data

### CODE REVIEW

**Comment by ersantosnet** [comment]
PR in testing..

File: db_assessment/optimusprime.py [hunk 1]
```diff
@@ -309,13 +332,35 @@ def argumentsParser():
 
     parser.add_argument("-importcomment", type=str, default='', help="Comment for the Import")
 
+
+    # SM: 04/28/2022:[Bug#90] Recreate views without loading the data:
+    parser.add_argument("-recreateviews", default=False, help="Recreate views without loading the data (Yes/No)")
```
Comment (ersantosnet) [inline db_assessment/optimusprime.py:337]: Should it be True or False?

Comment (ersantosnet) [reply db_assessment/optimusprime.py:337]: Instead of Yes/No because of 
==
default=False
==

Comment (smpawar) [reply db_assessment/optimusprime.py:337]: @sandeepmanocha @ersantosnet when I run it fails with below error:

```
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment/db_assessment$ python3 optimusprime.py -dataset bt_pull147 -fileslocation /Users/pawarsachin/Desktop/OP/Issues/beautifultb/op190_203   -projectname oraclepssupport -recreateviews Yes -dbversion 190 -collectionid ""
Traceback (most recent call last):
  File "optimusprime.py", line 391, in <module>
    runMain(args)
  File "optimusprime.py", line 82, in runMain
    if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
AttributeError: module 'import_db_assessment' has no attribute 'checkDataSetExists'
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment/db_assessment$ cd ..
(op-venv) ~/Desktop/OP/Issues/addflagsskipdataload/oracle-database-assessment$ grep checkDataSetExists */*
grep: db_assessment/__pycache__: Is a directory
grep: db_assessment/dbSQLCollector: Is a directory
grep: db_assessment/opConfig: Is a directory
db_assessment/optimusprime.py:        if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
```

Is there any code for 'checkDataSetExists' that got missed to be added to the commit?

**Comment by cofin** [comment]
Is it ok if we close this PR? I suspect there are quite a few merge conflicts to address now.

**Pull request closed without merging**